modules/test/playwright/tests/notification-web/userNotification.spec.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+import {readFileSync} from 'fs';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {notificationPagesTest} from '../../fixtures/notificationPagesTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import {blogsPagesTest} from '../../tests/blogs-web/fixtures/blogsPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	blogsPagesTest,
+	loginTest(),
+	notificationPagesTest,
+	workflowPagesTest
+);
+
+let assetType: string;
+let blogTitle: string;
+let workflowDefinitionId: number;
+let workflowDefinitionName: string;
+let workflowXMLDefinition: string;
+
+test.afterEach(
+	async ({
+		apiHelpers,
+		blogsPage,
+		configurationTabPage,
+		processBuilderPage,
+	}) => {
+		if (assetType && workflowDefinitionName) {
+			await processBuilderPage.goto();
+
+			await configurationTabPage.goTo();
```

Comment:
Hey Carlos, we can call the `await processBuilderPage.goto();` inside the `configurationTabPage.goTo();` funtion, to avoid calling these functions always together.